src/main/java/mate/controller/car/DeleteCarController.java

```diff
@@ -0,0 +1,24 @@
+package mate.controller.car;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.CarService;
+
+@WebServlet(urlPatterns = "/cars/delete")
+public class DeleteCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final CarService carService = (CarService) injector.getInstance(CarService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        Long carId = Long.valueOf(request.getParameter("id"));

```

Comment:
```suggestion
        Long id = Long.valueOf(request.getParameter("id"));
```
it's a controller for car's deletion, so it doesn't make sense to add car prefix to the variable's name

Reply:
✔️